x-pack/elastic-agent/docs/running-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[running-on-kubernetes]]
+[role="xpack"]
+=== Run {agent} on Kubernetes
+
+You can use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
```

Comment:
I'd delete this paragraph or we may risk to forget about it in the near future. In other words, docs shouldn't "warn" or "announce" future updates or changes

Reply:
I think we are covered by the `ifeval::["{release-state}"=="unreleased"]` :)
The same we do also in manifests for x-Beats too: https://github.com/elastic/beats/blame/master/metricbeat/docs/running-on-kubernetes.asciidoc#L9